registry/mainnet/interchain/squid.tokenlist.json

```diff
@@ -6169,8 +6169,7 @@
           "tokenManager": "0x382A35F2D2D80cB43A56857DdA2E8bc5bb3053cC",
           "tokenManagerType": "mintBurn"
         }
-      ],
-      "coinGeckoId": ""
```

Comment:
Why remove this? We always keep the field even if it's an empty string for clarity

Reply:
`coingeckoId` still exists for this asset as empty string just exists under deploy salt, unsure why the order of the keys got a bit jumbled on some of the asset. Will make sure to move the `coingeckoId` for this FURY token up to the correct spot.